### ISSUE TITLE
Changing keyboard layouts dynamically

### ISSUE THREAD

**Issue by milisarge** [CuarzoSoftware/Louvre#61]
Is it possible to change keyboard layout dynamically without restarting the compositor?

**Comment by Fox2Code** [comment]
Translation: Is there an easy to use API to change keyboard layout at runtime.

Also, yeah, I think keyboard layout switching is important, I have a French keyboard, and sometimes I have to switch keyboard layout around to help me make some games work correctly.

**Comment by ehopperdietzel** [comment]
> Is it possible to change keyboard layout dynamically without restarting the compositor?

Yes, [LKeyboard::setKeymap()](https://cuarzosoftware.github.io/Louvre/class_louvre_1_1_l_keyboard.html#af3224ac0e4648b52dbb8da1281b83549) can be called at any time and will update all clients keymaps.
